File: messages/src/main/java/com/thundashop/core/resturantmanager/RestaurantTableDay.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.thundashop.core.resturantmanager;

import com.thundashop.core.common.DataCommon;
import java.util.ArrayList;
import java.util.List;

/**
 *
 * @author ktonder
 */
public class RestaurantTableDay extends DataCommon {
    public String date;
    public String tableId; 
    public List<TableReservation> events = new ArrayList();
}

File: html5apps/terminal/events/core/resturantmanager/RestaurantTableDay.php
<?php
class core_resturantmanager_RestaurantTableDay extends core_common_DataCommon  {
	/** @var String */
	public $date;

	/** @var String */
	public $tableId;

	/** @var core_resturantmanager_TableReservation[] */
	public $events;

}
?>